dev/attest/src/fs.ts

```diff
@@ -27,22 +34,50 @@ export const ensureDir = (path: string) => {
 
 export const readFile = (path: string) => readFileSync(path).toString()
 
+export const readFileAsync = (
+    filePath: string,
+    onSuccess: (data: string) => void
+): void => {
+    readFile_(filePath, { encoding: "utf-8" }, (err, data) => {
+        if (err !== null) {
+            return console.error(
+                `Received an error while attempting to read file "${filePath}". \nError Received: \n${err}`
+            )
+        } else {
+            return onSuccess(data)
+        }
+    })
+}
```

Comment:
Using the async versions of `readFile` and `writeFile` simplified the implementation of `rewritePaths` significantly. I'd be hesitant to refactor `rewritePaths` to use the synchronous versions; although it would make our uses of the `fs` module more cohesive, it would an extra step to the build